eng/Versions.props

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <VersionPrefix>1.0.0</VersionPrefix>
+    <PreReleaseVersionLabel>prerelease</PreReleaseVersionLabel>
+    <MicrosoftDotNetCliUtilsPackageVersion>2.0.0</MicrosoftDotNetCliUtilsPackageVersion>
+    <SystemCollectionsPackageVersion>4.3.0</SystemCollectionsPackageVersion>
+    <SystemIOFileSystemPrimitivesPackageVersion>4.3.0</SystemIOFileSystemPrimitivesPackageVersion>
+    <MicrosoftWin32PrimitivesPackageVersion>4.3.0</MicrosoftWin32PrimitivesPackageVersion>
+    <SystemIOFileSystemPackageVersion>4.3.0</SystemIOFileSystemPackageVersion>
+    <SystemNetPrimitivesPackageVersion>4.3.0</SystemNetPrimitivesPackageVersion>
+    <runtimenativeSystemPackageVersion>4.3.0</runtimenativeSystemPackageVersion>
+    <SystemSecurityPrincipalPackageVersion>4.3.0</SystemSecurityPrincipalPackageVersion>
```

Comment:
Consider using `Version` instead of `PackageVersion` as a suffix - not that it really matters, but will be more consistent with https://github.com/dotnet/coreclr/pull/27606 for example.